workflows/sarek.nf

```diff
@@ -206,7 +206,7 @@ if (params.wes && !params.step == 'annotate') {
     else log.warn("Intervals file was provided without parameter `--wes`: Pipeline will assume this is Whole-Genome-Sequencing data.")
 } else if (params.intervals && !params.intervals.endsWith("bed") && !params.intervals.endsWith("list")) error("Intervals file must end with .bed, .list, or .interval_list")
 
-if (params.step == 'mapping' && params.aligner.contains("dragmap") && !(params.skip_tools && params.skip_tools.split(',').contains("baserecalibrator"))) {
+if (params.step == 'mapping' && params.aligner.contains("dragmap") && !(params.skip_tools && params.skip_tools.split(',').any{ toLowerCase().contains("baserecalibrator") } )) {
```

Comment:
In line 195, you wrote `.any{ it.toLowerCase().contains....`Here you write `.any{ toLowerCase().contains` without the `it`.  Are both ways of writing it okay?